### ISSUE TITLE
Resources get embedded into scene on load

### ISSUE THREAD

**Issue by KoBeWi** [godotengine/godot#64158]
### Godot version

3.x 1394ee9e5e89f87bace2aefccab117dc966e778f

### System information

Windows 10 x64

### Issue description

When you load a scene with external resources, they get converted to sub-resources (and in some cases it converts textures to images?).
```
[sub_resource type="StreamTexture" id=2]
flags = 4
load_path = "res://.import/icon.png-487276ed1e3a0c39cad0279d744ee560.stex"
```

### Steps to reproduce

1. Create a scene
2. Add Sprite2D with a texture
3. Reload the scene
4. Save
5. Check tscn file

**Comment by Razoric480** [comment]
I'll take a look tonight.

**Comment by akien-mga** [comment]
Fixed by #64200.
